prerelease-msvc/action.yaml

```diff
@@ -167,8 +166,19 @@ runs:
         echo "version=${{ steps.prerelease.outputs.new-release-version }}" >> $GITHUB_OUTPUT
         echo "run-url=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> "$GITHUB_OUTPUT"
 
+    - name: Check Docker Config File
+      id: check-docker-config
+      shell: bash
+      run: |
+        if [ ! -f "${{ inputs.docker-config-file }}" ]; then
+          echo "Docker config file not found: ${{ inputs.docker-config-file }}. Skipping Docker build."
+          echo "DOCKER_BUILD=false" >> $GITHUB_ENV
+        else
+          echo "DOCKER_BUILD=true" >> $GITHUB_ENV
+        fi
+
     - uses: open-turo/actions-jvm/build-docker@v1
-      if: steps.prerelease.outputs.new-release-published == 'true'
+      if: env.DOCKER_BUILD == 'true' && steps.prerelease.outputs.new-release-published == 'true'
```

Comment:
Maybe a simpler approach is: make the docker hub username and password optionals. If they are passed then it's assumed that an image needs to be built regardless of the existence of the docker file.

I believe front-end did something like this for their setup to make docker builds optional